src/Symfony/Component/Serializer/Tests/Fixtures/serialization.yml

```diff
@@ -16,6 +16,20 @@
       serialized_name: 'baz'
     bar:
       serialized_name: 'qux'
+'Symfony\Component\Serializer\Tests\Fixtures\SerializedNameWithGroupsDummy':
+  attributes:
+    foo:
+      serialized_name: 'baz'
+    foos:
+      serialized_name:
+        bazs: ~
+    barWithGroup:
+      serialized_name:
+        bargroups: ['group1']
+    quuxWithGroups:
+      serialized_name:
```

Comment:
should be `serialized_names`

Reply:
done but not sure if I still need to maintain serialized_name without s here...